test/e2e/test-che-page-info/test.mjs

```diff
@@ -1,55 +1,49 @@
-import { getStats, getDatasetItems, run, expect, validateDataset } from '../tools.mjs';
-
-await run(import.meta.url, 'cheerio-scraper', {
-    startUrls: [{
-        url: 'https://apify.com/store',
-        method: 'GET',
-        userData: { label: 'START' },
-    }],
-    keepUrlFragments: false,
-    pseudoUrls: [{
-        purl: 'https://apify.com/apify/web-scraper',
-        method: 'GET',
-        userData: { label: 'DETAIL' },
-    }],
-    linkSelector: 'a',
-    pageFunction: async function pageFunction(context) {
-        const { request, log, skipLinks, $ } = context;
+import { Actor } from 'apify';
+import { CheerioCrawler, pushData } from '@crawlee/cheerio';
+import { getDatasetItems, expect, validateDataset, skipTest, initialize } from '../tools.mjs';
+
+skipTest('Apify store lazy loads items now, which cannot be easily tested with Cheerio');
+
+await initialize(import.meta.url);
+
+const crawler = new CheerioCrawler({
+    ignoreSslErrors: false,
+    async requestHandler({ request, log, $, enqueueLinks }) {
         const { userData: { label } } = request;
 
         if (label === 'START') {
             log.info('Store opened!');
+            await enqueueLinks({
+                globs: [{ glob: 'https://apify.com/apify/web-scraper', userData: { label: 'DETAIL' } }],
+            });
         }
 
         if (label === 'DETAIL') {
             const { url } = request;
             log.info(`Scraping ${url}`);
-            await skipLinks();
 
             const uniqueIdentifier = url.split('/').slice(-2).join('/');
             const title = $('header h1').text();
             const description = $('header span.actor-description').text();
             const modifiedDate = $('ul.ActorHeader-stats time').attr('datetime');
             const runCount = $('ul.ActorHeader-stats > li:nth-of-type(3)').text().match(/[\d,]+/)[0].replace(/,/g, '');
 
-            return {
+            await pushData({
```

Comment:
interesting, there should be no `pushData` helper i'd say, only `Actor.pushData`

will remove the function myself, please use `Actor.pushData` here so things won't break because of it

(same for `openDataset`)

https://github.com/apify/apify-ts/commit/8ef77f54e0bf40a0e750fc78724e8dbc12f8bc14